docroot/themes/humsci/humsci_basic/src/js/tables/wrap.js

```diff
@@ -0,0 +1,8 @@
+// TODO: Can we just use the global jQuery?
+// $('table').wrap('<div class="hb-table-wrap"></div>');
+
+const el = document.querySelector('table');
+const wrapper = document.createElement('div');
+wrapper.className = 'hb-table-wrap';
```

Comment:
this is only getting added to one table on the page, should be all

Reply:
Updated to loop through all table elements.